mephisto/client/cli.py

```diff
@@ -319,9 +319,9 @@ def metrics_cli(args):
         shutdown_grafana_server,
     )
 
-    if len(args) == 0 or args[0] not in ["install", "launch", "cleanup"]:
+    if len(args) == 0 or args[0] not in ["install", "view", "cleanup"]:
         click.echo(
-            "Usage: mephisto metrics <install|launch|cleanup>\n"
+            "Usage: mephisto metrics <install|view|cleanup>\n"
             f"install: Installs prometheus and grafana to {METRICS_DIR}\n"
             f"view: Launches a Prometheus and Grafana server, and shuts down on exit\n"
             f"cleanup: Shuts down prometheus and grafana resources that may have persisted"
```

Comment:
small nit: inconsistent capitalization. Prometheus and Grafana not capitalized for `install`/`cleanup` but are for `view`